.gitignore

```diff
@@ -25,3 +25,6 @@ coverage.html
 # Wix
 *.wixobj
 *.wixpdb
+
+# Tarball file
+*.tar.gz
```

Comment:
See other comment about moving the generated tarball into the `dist` folder in which case you wouldn't need this as the `dist` directory is already ignored above